rudder-reports/SOURCES/Makefile

```diff
@@ -30,7 +30,8 @@ localdepends: ./rudder-sources ./rudder.conf
 	mv rudder-sources-*/ rudder-sources/
 
 ./rudder.conf: ./rudder-sources
-	cp rudder-sources/rudder-techniques/initial-promises/node-server/distributePolicy/rsyslog.conf/rudder.conf rudder.conf
+	cd rudder-sources/rudder-techniques/ && $(MAKE)
+	cp initial-promises/node-server/distributePolicy/rsyslog.conf/rudder-rsyslog-root.conf rudder.conf
```

Comment:
Shouldn't this be rudder-sources/rudder-techniques/initial-promises ? You've shortened the path, but I'm not sure why. If the "cd" is persistent across lines, you need to cd back, because the sed commands below won't work.